xml/src/test/java/org/ehcache/xml/provider/CacheEventDispatcherFactoryConfigurationParserTest.java

```diff
@@ -17,47 +17,35 @@
 package org.ehcache.xml.provider;
 
 import org.ehcache.config.Configuration;
-import org.ehcache.config.builders.ConfigurationBuilder;
 import org.ehcache.impl.config.event.CacheEventDispatcherFactoryConfiguration;
 import org.ehcache.spi.service.ServiceCreationConfiguration;
-import org.ehcache.xml.ConfigurationParser;
-import org.ehcache.xml.XmlConfigurationTest;
 import org.junit.Test;
 import org.xml.sax.SAXException;
 
 import java.io.IOException;
-import java.net.URL;
 
 import javax.xml.bind.JAXBException;
 import javax.xml.parsers.ParserConfigurationException;
 
-import static org.ehcache.config.builders.ConfigurationBuilder.newConfigurationBuilder;
-import static org.hamcrest.core.Is.is;
-import static org.hamcrest.core.IsInstanceOf.instanceOf;
-import static org.junit.Assert.*;
+import static org.assertj.core.api.AssertionsForInterfaceTypes.assertThat;
```

Comment:
Nice! the import should actually be `org.assertj.core.api.Assertions` which will alias at the right place :-)